Makefile

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/make -f
 
-PACKAGES_SIMTEST=$(shell go list ./... | grep '/simulation')
-VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
```

Comment:
`git describe`'s output does show commit information. What's wrong with that?

Reply:
ref https://git-scm.com/docs/git-describe

Reply:
I'm thinking of using the branch name instead of the tag since it avoids picking up an old tag (v3.0.0) on the main branch.

Reply:
Oh that's because we don't have annotated tags in `main`. Ok makes sense :+1: